packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -25,8 +25,8 @@ const initialDebugConfigurations = {
       name: 'Launch Apex Debugger',
       type: 'apex',
       request: 'launch',
-      userIdFilter: '',
-      requestTypeFilter: '',
+      userIdFilter: [],
+      requestTypeFilter: [],
```

Comment:
I'm confused. I thought you had this as an array for the completions of requestType filter a few PR's ago?
Was this just a change that we didn't catch? Was there no type/compile failure for the diverging types?

Reply:
Yea, we missed this one and no type/compile failure. It was used in a contribution point in package.json. We'll have to remove this soon because it'll be deprecated in an upcoming release.

`https://github.com/Microsoft/vscode/issues/33794`